### PR TITLE
ci: Travis -> GitHub Actions

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by odscjames** [comment]
There are 2 broken runs as well.

**Comment by odscjames** [comment]
[Switching from Travis to GH Actions](https://trello.com/c/oz6updTN/127-switching-from-travis-to-gh-actions)

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+        architecture: x64
+    # This is from https://github.com/elastic/elastic-github-actions/tree/master/elasticsearch
```
Comment (michaelwood) [inline .github/workflows/test.yml:14]: What about using something like https://github.com/marketplace/actions/setup-elasticsearch  ?

Comment (odscjames) [reply .github/workflows/test.yml:14]: Ah, didn't think to check unofficial ones. I checked the official one and that says "DONT USE THIS YET". https://github.com/elastic/elastic-github-actions/tree/master/elasticsearch . I wouldn't rush to use an unofficial one myself.

Comment (michaelwood) [reply .github/workflows/test.yml:14]: Fair enough

**Review by michaelwood — APPROVED** [review]

**Pull request merged**